### ISSUE TITLE
Compile error rs_stats

### ISSUE THREAD

**Issue by hojjatjafary** [librsync/librsync#77]
Hi,
I wrote a simple code that only calls "rs_sig_file", but I get this compile error:
error C3646: 'start': unknown override specifier
the 'start' is a member of "rs_stats" inside the "librsync.h", but when I include <time.h> or <ctime> above the "librsync.h" include then the project compiles successfully.
Why I need to include time library inside my own cpp file that uses only a function call of "rs_sig_file", I also tried to include "stdlib.h" that is works too.
I compile using Visual studio 15.

```
#include <stdlib.h>
//#include <ctime>
#include "rsync/librsync.h"
#include "sync_signature.hpp"

int SyncSignature::generateSignature(std::string inputFilePath, std::string sigFilePath)
{
    FILE *fpInput;
    fpInput = fopen(inputFilePath.c_str(), "rb");

    FILE *fpSig;
    fpSig = fopen(sigFilePath.c_str(), "wb+");

    // The MD4 sig magic is deprecated.
    rs_result res = rs_sig_file(fpInput, fpSig, 2048, 32, RS_BLAKE2_SIG_MAGIC, NULL);

    fclose(fpInput);
    fclose(fpSig);

    return 0;
}
```

**Comment by sourcefrog** [comment]
Yes it seems like `librsync.h` ought to include `time.h`

**Comment by dbaarda** [comment]
I've included this fix in pull request #104. I'll merge it if I don't get any complaints in a day or so.

**Comment by dbaarda** [comment]
This should be fixed now. Please reopen if you find otherwise.
